jina/executors/indexers/__init__.py

```diff
@@ -270,7 +289,7 @@ def add(self, keys: Iterable[str], vectors: 'np.ndarray', *args, **kwargs) -> No
         raise NotImplementedError
 
     def query(
-        self, vectors: 'np.ndarray', top_k: int, *args, **kwargs
+        self, vectors: 'EncodingType', top_k: int, *args, **kwargs
```

Comment:
would it be more clear if we creatr anotjher type var saying EncodeBatchType that is just an alias?